beesdoo_shift_swap/models/subscribed_underpopulated_shift.py

```diff
@@ -0,0 +1,282 @@
+from odoo import models, fields, api
+from datetime import datetime, timedelta, date
+from odoo.exceptions import Warning
+
+def daterange(start_date, end_date):
+    for n in range(int ((end_date - start_date).days)) :
+        return start_date + timedelta(n)
+
+
+class subscribe_underpopulated_shift(models.Model):
+    _name = 'beesdoo.shift.subscribed_underpopulated_shift'
+
+
+    def _get_selection_status(self):
+        return [
+            ('draft','Draft'),
+            ('validate','Validate'),
+            ('done','Done')
+        ]
+
+    state=fields.Selection(
+        selection=_get_selection_status,
+        default='draft'
+    )
+    worker_id = fields.Many2one(
+        "res.partner",
+        domain=[
+            ("is_worker", "=", True),
+            ("working_mode", "in", ("regular", "irregular")),
+            ("state", "not in", ("unsubscribed", "resigning")),
+        ],
+    )
+    exchanged_timeslot_id = fields.One2many(
+        comodel_name='beesdoo.shift.timeslots_date',
+        inverse_name='id',
+        string='exchanged_shift'
+    )
+    exchange_status = fields.Boolean(default=False, string="status exchange shift")
```

Comment:
For name in `string` attribute capitalise each words: `string="Status Exchange Shift"`.